requirements/base.in

```diff
@@ -0,0 +1,8 @@
+# Core requirements for using this package template
+-c constraints.txt
+
+Django
+PyJWT
+six
+social-auth-app-django
+social-auth-core
```

Comment:
You could omit this one, social-auth-app-django declares it as a dependency so it'll get pulled in anyway and I don't think we use it directly.

Reply:
It looks like `auth_backends/strategies.py` might use it directly (`social_core` around lines 20–30)—does that change anything in your estimation?

Reply:
Sure, go ahead and leave it with a comment at the end of the line explaining briefly what we directly use it for.